pkg/virt-launcher/virtwrap/network/common_test.go

```diff
@@ -106,18 +106,29 @@ var _ = Describe("Common Methods", func() {
 var _ = Describe("VIF", func() {
 	Context("String", func() {
 		It("returns correct string representation", func() {
```

Comment:
I guess the name should now mention it is IPv4.

Reply:
it is not ipv4. It contains ipv6 as well, just in this case the ipv6 is nil.